site/content/blogs/release-v126.en.md

```diff
@@ -36,6 +36,90 @@ Copilot v1.26 brings several new features and improvements:
 
 ## `storage init` for environment addons
 
+Previously, `copilot storage init` only supported storage addons attached to workloads: you need to run 
+`copilot svc deploy` in order to deploy the storage, and the storage is deleted along with the service
+when you run `copilot svc delete`.
+
+Now, you have the option to create environment-level storage addons: the storage is deployed when you run `copilot env deploy`,
+and isn't deleted until you delete the whole environment by running `copilot env delete`.
+
+Similar to the workload-level storage, the environment-level storage is, under the hood, just another [environment addon](../docs/developing/addons/environment.en.md)!
+
+### Best Practice By Default
+Following the best practice in the microservice world, Copilot encourages you to set up storages that are each accessible
+by only one service, instead of monolith storages that are shared by all microservices. Therefore, Copilot assumes
+that your storage is designed to be accessed by one of your services or jobs, even when it is meant to live and die
+with the environment. 
```

Comment:
```suggestion
that your storage is designed to be accessed by one of your services or jobs, even when it shares the same lifecycle with the environment. 
```

Reply:
I removed the whole clause about "even when it shares the same lifecycle with the environment.". Figured it might help nothing but adding confusion